packages/debug/src/browser/debug-session-connection.ts

```diff
@@ -243,14 +243,22 @@ export class DebugSessionConnection implements Disposable {
         const connection = await this.connectionPromise;
         const messageStr = JSON.stringify(message);
         if (this.traceOutputChannel) {
-            this.traceOutputChannel.appendLine(`${this.sessionId.substring(0, 8)} theia -> adapter: ${messageStr}`);
+            const now = new Date();
+            /// offset in milliseconds
+            const tzoffset = now.getTimezoneOffset() * 60 * 1000;
+            const dateStr = new Date(now.getTime() - tzoffset).toISOString();
```

Comment:
Suggestion: As far as I know, `toISOString` will always convert the input time into UTC. So we can simplify this by using the following code. The same applies to the other change.
```suggestion
            const dateStr = new Date().toISOString();
```

Reply:
minus the timezone offset for the output to be the same as local time. If not, the time string should be `2021-11-27T02:40:06.428Z` for a UTC+9 which should be `2021-11-27 11:40:06.428` as local. It is good, but it looks strange. So I make a new date minus the timezone offset to create a data string

Reply:
I update to use `toLocaleString`